pkg/capr/planner/planner.go

```diff
@@ -166,6 +184,7 @@ func New(ctx context.Context, clients *wrangler.Context, functions InfoFunctions
 			secretCache: clients.Core.Secret().Cache(),
 		},
 		retrievalFunctions: functions,
+		Equalities:         equalities,
```

Comment:
`equality.Semantic.DeepEqual` is used in a few different places too (e.g. `minorPlanChangeDetected`). Are we going to eventually just move towards only using this version of `equalities` throughout the planner? 

The standard `DeepEqual` doesn't marshal to json, so there would be a performance benefit in only using this when handling plan / file objects (or just general objects which contain functions that we should ignore)

Reply:
Updated to not use `json.Marshal`. FWIW it only needs to be used in when an `plan.NodePlan` is involved (or more specifically, a `plan.File`). At some point I'll probably refactor this to pull it out of the struct.